ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -1943,7 +2013,7 @@ presubmits:
     name: pull-ci-openshift-installer-master-e2e-metal-single-node-live-iso
     optional: true
     rerun_command: /test e2e-metal-single-node-live-iso
-    run_if_changed: ^([^d]|d(d|o(d|cd))*([^do]|o([^cd]|c[^ds])))*(d(d|o(d|cd))*(oc?)?)?$
+    skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(aws|azure|gcp|kubevirt|libvirt|openstack|ovirt|vsphere)/)
```

Comment:
This and many other jobs in this PR are `always_run: false`. Won't this change remove the trigger, so that the job will never be run automatically? Perhaps I'm missing something, but it seems like we also need to change this to `always_run: true`.

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#triggering-jobs-based-on-changes

Reply:
Oh, cool. I find the name and even the explanation in the docs misleading, but the example clears it up for me. Thanks for explanation.

Reply:
Yeah, it's super confusing and [I had the same question myself](https://github.com/openshift/release/pull/20579#discussion_r677938225) the first time I saw it, but basically you can set `always_run` OR `run_if_changed` OR `skip_if_only_changed`.